From 5efb1d9dbe156738385a98f9e3f90eec050f4eaa Mon Sep 17 00:00:00 2001
From: Pardha Kesapragada <pkesapragada@habana.ai>
Date: Wed, 17 Feb 2021 19:29:20 +0200
Subject: [PATCH] [SW-20154] add per platform makefiles

Gaudi2/Greco will have individual compiler and
linker options, which will be added in subsequent
patches. Hence add support for per platform makefiles

Change-Id: I217cebc738aafd1708eac49fb65335aba057cbf5
Signed-off-by: Pardha Kesapragada <pkesapragada@habana.ai>
---

diff --git a/Makefile b/Makefile
index 56cc148..2cade4d 100644
--- a/Makefile
+++ b/Makefile
@@ -40,6 +40,8 @@
 CFLAGS = -DELP_ARCV2
 export CFLAGS += -Os -fdata-sections -ffunction-sections
 
+include $(HL_DEV)/Makefile
+
 ARCBOOT_COMMON_SRCS_C := $(shell ls common/*.c common/rsa/*.c $(HL_DEV)/$(HL_TARGET).c)
 ARCBOOT_COMMON_SRCS_C += $(shell ls $(HL_DEV)/*.c)
 
diff --git a/gaudi2_arc/Makefile b/gaudi2_arc/Makefile
new file mode 100644
index 0000000..e69de29
--- /dev/null
+++ b/gaudi2_arc/Makefile
diff --git a/goya2_arc/Makefile b/goya2_arc/Makefile
new file mode 100644
index 0000000..e69de29
--- /dev/null
+++ b/goya2_arc/Makefile
